src/stories/drawer/Drawer.module.scss

```diff
@@ -26,51 +26,56 @@ $panelAnimationDelay: var(--pte-animations-duration-fast);
     position: absolute;
     inset: 0;
 
-    &.overlayBlured {
+    &.overlayBlur {
         backdrop-filter: blur(0);
-        background-color: var(--pte-colors-backgroundOverlayDark);
+        background-color: var(--pte-new-colors-overlayPageBackground);
         will-change: backdrop-filter, opacity;
     }
 
     &.overlayGreyed {
```

Comment:
Made this name swap for the overlays. Currently the `overlayStyle` prop values are either "greyed" or "blur", should that also switch to just "grey", or should I not because we used "greyed" in previous versions and renaming could break something?

Reply:
good catch -- let's switch that to grey as well to match. please review usages in midnights to ensure no breakage